### DISCUSSION TITLE
Downloading 下载过程中发现git地址不存在这种怎么处理那

### DISCUSSION THREAD

**Discussion by wzjaa** [tw93/Pake#480] [Q&A]
`> pake-cli@2.1.0 build
> npm run tauri build --release

> pake-cli@2.1.0 tauri
> tauri build

   Compiling app v0.1.0 (E:\tool\nvm\nvm\v16.20.0\node_global\node_modules\pake-cli\src-tauri)
    Finished release [optimized] target(s) in 33.05s
        Info Verifying wix package
 Downloading https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311-binaries.zip
       Error failed to bundle project: `https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311-binaries.zip: Connection Failed: Connect error: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 (os error 10060)`
file:///E:/tool/nvm/nvm/v16.20.0/node_global/node_modules/pake-cli/dist/cli.js:370
                reject(new Error(`${code}`));
                       ^

Error: 1
    at file:///E:/tool/nvm/nvm/v16.20.0/node_global/node_modules/pake-cli/dist/cli.js:370:24
    at E:\tool\nvm\nvm\v16.20.0\node_global\node_modules\pake-cli\node_modules\shelljs\src\exec.js:145:9
    at ChildProcess.exithandler (node:child_process:410:5)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1100:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:304:5)`

**Comment by wzjaa** [comment]
这个git地址是不存在的，该如何修改那

**Reply by WitherZuo** [reply]
应该是因为国内裸连访问 GitHub 失败导致的，可以考虑特殊方式上网连接 GitHub 重试

**Comment by wzjaa** [comment]
找到原因了，没有按照提供的文档安装WebView2
